pkg/parsers/kernel/kernel_test_unix.go

```diff
@@ -19,6 +21,7 @@ func assertParseRelease(t *testing.T, release string, b *VersionInfo, result int
 	}
 }
 
+// TestParseRelease tests the ParseRelease() function
```

Comment:
see my comment below, I would have use another name as there is a buildflag, and thus no need to lint it (I think :sweat_smile:).